### DISCUSSION TITLE
email attribute missing on token?

### DISCUSSION THREAD

**Discussion by astriffe** [zitadel/zitadel#2050] [Q&A] [answered]
Hi caos!

Might it be that the preferred email vanished from the token, while the checkbox 'User Info im ID Token' is still set?

Of course I might be wrong, but I can't find the email claim on any of the tokens I receive :)

Cheers, Alex

**Comment by fforootd** [comment, marked as answer]
Just for easy reference https://github.com/caos/zitadel/issues/2051 😁
